oxli/functions.py

```diff
@@ -112,14 +112,13 @@ def optimal_args_output_gen(unique_kmers, fp_rate):
     return "\n".join(to_print)
 
 
-def do_sanity_checking(args, desired_max_fp):
+def check_fp_rate(args, desired_max_fp):
     """
     simple function to check if the restrictions in the args (if there are any)
```

Comment:
(please also update this - it's not simple anymore :) and also is currently limited to looking at unique_kmers.

Also, you can deindent most of the function by changing that first 'if' to

```
if not args.unique_kmers:
   return